src/index.js

```diff
@@ -6,6 +6,7 @@ import range from './range';
 import switchMap from './switchMap';
 import tapError from './tapError';
 import toArray from './toArray';
+import concatArray from './concatArray';
```

Comment:
Imports, and exports should be alphabetical. Makes it easier to organise and determine what has or hasn't been imported.